cohesity_management_sdk/models/blackout_period.py

```diff
@@ -27,7 +27,7 @@ class BlackoutPeriod(object):
 
     # Create a mapping from Model property names to API property names
     _names = {
-        "Id":'id',
+        "id":'Id',
```

Comment:
Is this human err or generated from sdk ?

Reply:
humar error